modules/exploits/linux/http/unraid_eval.rb

```diff
@@ -0,0 +1,95 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::PhpEXE
+
+  Rank = ExcellentRanking
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'        => 'Unraid 6.8.0 Remote Code Execution',
+        'Description' => %q(
+          This module exploits two vulnerabilities affecting Unraid 6.8.0.
+          An authenticated bypass is used to gain access to the administrative
+          interface, and an insecure use of the extract PHP function can be abused
+          for arbitrary code execution as root.
+        ),
+        'Author'      =>
+          [
+            'Nicolas CHATELAIN <n.chatelain@sysdream.com>'
+          ],
+        'References'  =>
+          [
+            [ 'CVE', '2020-5847' ],
+            [ 'CVE', '2020-5849' ],
+            [ 'URL', 'https://sysdream.com/news/lab/2020-02-06-cve-2020-5847-cve-2020-5849-unraid-6-8-0-unauthenticated-remote-code-execution-as-root/' ],
+            [ 'URL', 'https://forums.unraid.net/topic/88253-critical-security-vulnerabilies-discovered/' ]
+          ],
+        'License'        => MSF_LICENSE,
+        'Platform'       => ['php'],
+        'Privileged'     => false,
```

Comment:
If the module gains code execution as `root`, then `Privileged` is `true`.

```suggestion
        'Privileged'     => true,
```